jvm-libs/linea/besu-rlp-and-mappers/src/main/kotlin/linea/domain/TransactionTypeMapper.kt

```diff
@@ -0,0 +1,23 @@
+package linea.domain
+
+import org.hyperledger.besu.datatypes.TransactionType
+
+fun TransactionType.toDomain(): linea.domain.TransactionType {
+  return when (this) {
+    TransactionType.FRONTIER -> linea.domain.TransactionType.FRONTIER
+    TransactionType.EIP1559 -> linea.domain.TransactionType.EIP1559
+    TransactionType.ACCESS_LIST -> linea.domain.TransactionType.ACCESS_LIST
+    TransactionType.BLOB -> linea.domain.TransactionType.BLOB
```

Comment:
Blobs and delegate code isn't really a part of our domain. Isn't it better to throw right here in case we get it from Besu? I also think that we should remove them from `TransactionType` so compiler helps those who is less familiar with the code to know what's supported and what is not

Reply:
I prefer to have them here and throw an exception on the block creation monitor because we cannot conflate them. If for any reason we can one sequenced we will see clearly why it failed. 

Although, I don't have the time to implement 100% completeness of all cases (tx serialization, etc), but having all transaction types in the enum is valuable IMO because it raises awaress about those cases.

Reply:
Ok, up to you